src/amplitude-wrapper.js

```diff
@@ -144,6 +144,12 @@ import { version } from '../package.json';
   };
 
   var init = function(client, args) {
+      const argsLength = args.length;
+      const configuration =  args[argsLength - 1];
+      const uaParser = configuration['uaParser'];
```

Comment:
i would suggest that we call it something around user agent enrichment since that's the public facing feature or the value it brings. the ua parser is an implementation detail